### DISCUSSION TITLE
How to Block All Registries in Podman?

### DISCUSSION THREAD

**Discussion by Tinkotsu** [containers/podman#23139] [Q&A] [answered]
Hello,

I am trying to configure Podman to block all registries by default using the registries.conf file. My goal is to prevent Podman from pulling images from any registry unless explicitly allowed.

Here's what I have tried so far:

```
[[registry]]
location="*"
prefix=""
blocked=true
```

Unfortunately, this configuration does not block any registries.

**Comment by Tinkotsu** [comment, marked as answer]
I found policy.json configuration. Is this the correct way to isolate podman from the internet to keep registries.conf empty and configure policy.json like this?

policy.json
`{
    "default": [
        {
            "type": "reject"
        }
    ]
}`

**Reply by rhatdan** [reply]
Yes
